documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,132 @@
+
+***
+
+**Q:** For binding EJBs (presentation/business-tier), are unique and/or dynamic domain-names used?
+
+**A:** We do not enforce unique domain names. If you deploy two domains that must interoperate using RMI/EJB/JMS/JTA/and such, or that share RMI/EJB/JMS/JTA/and such clients, which concurrently communicate with both domains, then, as usual, the domain names must be configured to be different (even if they have different `domain-uids`).
+
+***
+
+#### Load Balancers
+
+**Q:** Load balancing and failover inside a DataCenter (HTTPS and T3s)?
+
+**A:** We originally certified on Traefik with the Kubernetes cluster; this is a very basic load balancer.  We have also certified other more sophisticated load balancers. See [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}).
+
+***
+
+#### Life Cycle and Scaling
+
+**Q:** How to deal with grow and shrink? Cluster and non-cluster mode.
+
+**A:** You can scale and shrink a configured WebLogic cluster (a set of preconfigured Managed Servers) or a dynamic WebLogic cluster (a cluster that uses templated Managed Servers) using different methods. See [Scaling]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling.md" >}}).
+* Manually, using Kubernetes command-line interface, `kubectl`.
+* WLDF rules and policies; when the rule is met the Administration Server sends a REST call to the operator which calls the Kubernetes API to start a new pod/container/server.
+* We have developed and made open source the [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter) which exports WebLogic metrics to Prometheus and Grafana.  In Prometheus, you can set rules similar to WLDF and when these rules are met, a REST call is made to the operator which invokes a Kubernetes API to start a new pod.
+
+
+**Q:** Container life cycle: How to properly spin up and gracefully shut down WLS in a container?
+
+**A:** The operator manages container/pod/WebLogic Server life cycle automatically; it uses the Node Manager (internally) and additional approaches  to do the following operations:
+* Entrypoint - start WebLogic Server.
+* Liveliness probe – check if the WebLogic Server is alive.
+* Readiness probe – query if the WebLogic Server is ready to receive requests.
+* Shutdown hook – gracefully shut down WebLogic Server.
+
+These operations also can be done manually using the Kubernetes command-line interface, `kubectl`.
+
+For more information, see the [Domain life cycle]({{< relref "/userguide/managing-domains/domain-lifecycle/_index.md" >}}) documentation.
+
+***
+
+
+#### Patching and Upgrades
+
+**Q:** Patching: rolling upgrades, handling of one-off-patches and overlays, CPUs, and such.
+
+**A:** For relevant information, see [Apply patched images to a running domain] , [Rolling restarts]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting#overview" >}}), and [CI/CD considerations]({{< relref "/userguide/cicd/_index.md" >}}).
```

Comment:
Placeholder: Missing relref for `[Apply patched images to a running domain] ` (which I assume is intended to reference the new patching document that coming in pull 2328).

Reply:
Correct.